{{ cookiecutter.repo_name }}/.isort.cfg

```diff
@@ -0,0 +1,4 @@
+[settings]
+line_length = 115
```

Comment:
115 not 120?

Reply:
I've used the default for the [`bluesky-queueserver` repo](https://github.com/bluesky/bluesky-queueserver/blob/c19ea0efe92dc7be21b0c87693df923cb4cc58da/.flake8#L11). I am not sure where it came from originally.

Please note, the value appears in three files:
- `pyproject.toml`
- `.flake8`
- `.isort.cfg`

Reply:
xref with the note about the decision: https://github.com/bluesky/bluesky-queueserver/pull/53.

Reply:
Good enough rationale for me. If a dev wants to change it, they certainly can.